### DISCUSSION TITLE
How to integrate the keyclock as single sign on with appwrite.

### DISCUSSION THREAD

**Discussion by ansulagrawal** [appwrite/appwrite#9045] [Q&A]
I have project where I use keyclock as authentication and node server and react frontend. I don't want user to login in appwrite now, I want to use single sign on b/w appwrite and keyclock. I can see multiple oauth but didn't find the keyclock in it.

**Comment by DH-555** [comment]
Make an issue with that feature request. It will be appreciated. Thanks!
